features/ui/input/Input.tsx

```diff
@@ -0,0 +1,56 @@
+import { DetailedHTMLProps, InputHTMLAttributes } from "react";
+import style from "./input.module.scss";
+
+interface InputProps {
+  error?: boolean;
+  errorText?: string;
+  label?: string;
+  hint?: string;
+  icon?: string;
+  ariaText: string;
+}
+
+interface InputAttributes
+  extends DetailedHTMLProps<
+    InputHTMLAttributes<HTMLInputElement>,
+    HTMLInputElement
+  > {}
+
+export const Input = ({
+  error,
+  label,
+  hint,
+  icon,
+  errorText,
+  ariaText,
+  ...inputProps
+}: InputAttributes & InputProps) => {
+  return (
+    <label
+      aria-label={!label ? ariaText : ""}
```

Comment:
```suggestion
      aria-label={label ? "" : ariaText}
```

It's kinda 🌵 a nitpick but I personally try to avoid inverting checked values because of the additional cognitive overhead of having to read lines of code while keeping track of things like "if this is true then it's actually false"

Reply:
I agree with this, sometimes I invert the checks mistakenly! Will be committed.